devops/openshift/api-deploy.yml

```diff
@@ -236,21 +238,21 @@ objects:
         apiVersion: apps.openshift.io/v1
         kind: DeploymentConfig
         name: ${NAME}
-      minReplicas:  "${{REPLICAS}}"
+      minReplicas: "${{REPLICAS}}"
       maxReplicas: 10
       metrics:
-          - type: Resource
-            resource:
-              name: cpu
-              target:
-                type: Utilization
-                averageUtilization: 80
-          - type: Resource
-            resource:
-              name: memory
-              target:
-                type: AverageValue
-                averageValue: "230M"
+        - type: Resource
```

Comment:
Is there a formatting change here?

Reply:
On saving the file, the editor auto-formats.